src/sections/Partners/Partners.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import Section from "../../components/Section";
+import Title from '../../components/Title/Title';
+import Icon, {
+  apps
+} from "../../components/Icon";
+import styles from "./Partners.module.scss";
+
+const partnersLinksConfig = [
+  {
+    name: "Logobrand",
```

Comment:
I think that the images of partner's logos could include names of their companies. In that case, we may not need the text in that case - the text from attached image was part of the logos 🤔  Let's wait for the assets and get back to the topic 😉

Reply:
Deleted the name but it's still used in tooltip (which is not really necessary) and in alt text.